src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCaseUpdater.java

```diff
@@ -116,17 +120,22 @@ public ProcessResult updateCase(
                 );
                 return new ProcessResult(updateResponse.warnings, emptyList());
             } else {
-                updateCaseInCcd(
+                Either<List<String>, Boolean> updateResult =  updateCaseInCcd(
                     configItem.getService(),
                     ignoreWarnings,
                     idamToken,
                     s2sToken,
                     userId,
+                    existingCaseId,
                     exceptionRecord,
                     updateResponse.caseDetails,
                     startEvent
                 );
 
+                if (updateResult.isLeft()) {
```

Comment:
what does updateResult.isLeft() = true means ?

Reply:
It means 'not right' ie. wrong

Reply:
a comment would be nice imo